resource-service/src/main/java/uk/gov/gchq/palisade/service/resource/request/GetResourcesByIdRequest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2019 Crown Copyright
```

Comment:
Copyright not changed as class just moved from common.
Will update if desired

Reply:
I would be inclined to update it, along with all the other GetResourcesByXXX classes that were moved from common.